common/test/unit/com/thoughtworks/go/helper/StageMother.java

```diff
@@ -231,6 +233,20 @@ public static Stage cancelledStage(String stageName, String jobName) {
         return stage;
     }
 
+    public static StageInstanceModel toStageInstanceModel(Stage stage) {
```

Comment:
Move this to it's own StageInstanceModelMother with a descriptive method on what it does 
Refer https://github.com/gocd/gocd/blob/22492eec4137f160f3054b4e5b9e300388023e37/server/test/unit/com/thoughtworks/go/server/ui/helper/AgentsViewModelMother.java